test/core.controller.tests.js

```diff
@@ -480,23 +471,27 @@ describe('Chart.Controller', function() {
 	});
 
 	describe('controller.destroy', function() {
-		it('should remove the resizer element when responsive: true', function() {
+		it('should remove the resizer element when responsive: true', function(done) {
 			var chart = acquireChart({
 				options: {
 					responsive: true
 				}
 			});
 
-			var wrapper = chart.chart.canvas.parentNode;
-			var resizer = wrapper.firstChild;
+			setTimeout(function() {
```

Comment:
What is the purpose of the `setTimeout(, 0)` here? Ideally I'd love to test this without needing to be async and without needing to be at the back of the message queue